.github/workflows/codspeed.yml

```
@@ -45,10 +45,7 @@ jobs:
           cache: 'pnpm' # https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies
       - run: pnpm install --frozen-lockfile
       # Ensure we are running on the prod version of our libs
-      # Only charts and license are needed for the benchmarks
-      - run: pnpm --filter @mui/x-license build
-      - run: pnpm --filter @mui/x-charts build
-      - run: pnpm --filter @mui/x-charts-pro build
+      - run: pnpm build:codesandbox
```

WDYT about adding only the missing `@mui/x-internals` package? 🤔 
```suggestion
      - run: pnmp --filter @mui/x-internals build
      - run: pnpm --filter @mui/x-license build
      - run: pnpm --filter @mui/x-charts build
      - run: pnpm --filter @mui/x-charts-pro build
```

---

Could be an option, only issue is that it would break again if a new package is introduced in the future 😅 

Do you think it is worth it? Would cut like 1:30m from run time 🤔

---

I would say—yes. The time saved isn't negligible. 🤔 
Hopefully next time we remember that once a package is introduced to charts, it needs to be added to the benchmark as well. 🤞🙈